modules/social_features/social_topic/social_topic.module

```diff
@@ -39,10 +40,12 @@ function social_topic_preprocess_node(array &$variables) {
     $topic_type = $node->get('field_topic_type');
     $topic_type_entities = $topic_type->referencedEntities();
     if (count($topic_type_entities) === 1) {
+      $curr_langcode = \Drupal::languageManager()->getCurrentLanguage(LanguageInterface::TYPE_CONTENT)->getId();
       foreach ($topic_type_entities as $topic) {
         $variables['metadata'] = t('in @topic', ['@topic' => $topic->link()]);
         // Set topic type link.
         $topic_type_url = Url::fromRoute('view.latest_topics.page_latest_topics', ['field_topic_type_target_id' => $topic->id()]);
+        $topic->isTranslatable() ? $topic = $topic->getTranslation($curr_langcode) : NULL;
```

Comment:
I think ternary expressions for an assignment are fine but I think we should avoid them for performing logic because they can reduce readability. Expanding this to an `if` statement is probably clearer.

```suggestion
        if ($topic->isTranslatable()) {
          $topic = $topic->getTranslation($curr_langcode);
        }
```

Reply:
changed that based on your suggestoin